solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -336,8 +337,8 @@ public void run() {
               for (Pair<String, byte[]> head : queue) {
                 byte[] data = head.second();
                 final ZkNodeProps message = ZkNodeProps.load(data);
-                if (log.isDebugEnabled()) {
```

Comment:
why is this made into `INFO` ? that can make the logs too verbose

Reply:
This was made INFO level because it's helpful for debugging Solr admin issues. It's only one log per operation, so it shouldn't be too verbose, and on our fork in practice it hasn't been an issue. (same goes for the other log made INFO level)

Reply:
We just need to log once when the overseer starts up , not for every operation.

Reply:
That makes sense to me, @hiteshk25 do you have any thought on this?

Reply:
Ah okay @magibney, that makes sense, I'll remove the logging stuff from this PR and we can take another look at that later.